test/router.ts

```diff
@@ -0,0 +1,317 @@
+var assert = require("assert");
+var router = require("../lib/router");
+
+describe("router.Router", () => {
+  describe("constructor", () => {
+    it("parses well formatted configs", () => {
+      process.env.SCHOOL = "Hogwarts";
+      const conf = `
+routes:
```

Comment:
It's actually going to be in a separate .kv_config (actual name pending) file.  Extending the launch yaml format was considered but ultimately rejected.  The top level `routes` key was added to make the file format future proof.  For example, we may add field-types or custom document definitions to .kv_config one day.